Dockerfile

```
@@ -14,33 +14,30 @@
 # limitations under the License.
 #
 
-FROM quay.io/codait/max-base:v1.3.2
+FROM quay.io/codait/max-base:v1.4.0
 
 ARG model_bucket=https://max-cdn.cdn.appdomain.cloud/max-breast-cancer-mitosis-detector/1.0.1
 ARG model_file=assets.tar.gz
 
-RUN apt-get update && apt-get install -y libopenslide0 gcc && rm -rf /var/lib/apt/lists/*
-
-WORKDIR /workspace
+RUN sudo apt-get update && sudo apt-get install -y libopenslide0 gcc && sudo rm -rf /var/lib/apt/lists/*
 
 RUN wget -nv --show-progress --progress=bar:force:noscroll ${model_bucket}/${model_file} --output-document=assets/${model_file} && \
   tar -x -C assets/ -f assets/${model_file} -v && rm assets/${model_file}
 
-COPY requirements.txt /workspace
+COPY requirements.txt .
 RUN pip install -r requirements.txt
 
-RUN cd /workspace && \
-    git clone https://github.com/codait/deep-histopath && \
+RUN git clone https://github.com/codait/deep-histopath && \
     cd deep-histopath && \
     git checkout c8baf8d47b6c08c0f6c7b1fb6d5dd6b77e711c33 && \
-    cd ../ && \
-    cp -R deep-histopath/. .
+    cd && \
```

This might be better accomplished with pushd:

```
pushd deep-histopath
git checkout ....
popd
```

---

Looks like it might be easier to just use `cd`
https://stackoverflow.com/a/5193087

---

OK, maybe the following is better than hardcoding home directory?
```suggestion
    cd - && \
```